packages/fast-development-site-react/src/components/shell/action-bar.tsx

```diff
@@ -11,9 +11,7 @@ export interface IShellActionBarManagedClasses {
 
 const style: ComponentStyles<IShellActionBarManagedClasses, IDevSiteDesignSystem> = {
     shell__action_bar: {
-        background: (config: IDevSiteDesignSystem): string => {
-            return config.lightGrey;
-        },
+        background: "#FFFFFF",
         padding: (config: IDevSiteDesignSystem): string => {
             return `${config.navigationBarHeight / 4}px`;
```

Comment:
Should be using `toPx()`

Reply:
I think that is outside of the scope of this PR. Just chatted with @nicholasrice and we need to think about how we will pull utilities into this project. If we reference fast-components-styles-msft we will run into circular dependency issues. Perhaps we need to move utilities into its own package...

Reply:
I have a task for this and will put it up for PR today

Reply:
Sure - I think having a utilities package makes sense